scripts/controllers/edit_button_controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Controller } from '@hotwired/stimulus'
+
+class EditButtonController extends Controller {
+
+  handleClick() {
+    fetch(
+      window.location.pathname + '?edit=1',
+      {
+        method: 'GET',
+      }
+    )
+  }
+
```

Comment:
this seems... kinda overkill for just `href="?edit=1"` ?

Reply:
Haha, yes it does. I may have I gotten so caught up porting that I didn't analyze the original code's usefulness. Here's the original. 

```
  $(document).on('click', '#edit-button', function () {
    $.ajax({
      method: 'GET',
      url: window.location.pathname,
      data: {
        edit: 1, // purge and keep out of cache temporarily
      }
    }) // no callback on this.
  })
```

I don't really get the code comment: Purge what? Why keep out of cache? Why temporarily? I think we can just use a regular link, as you suggested, unless this code snippet changes your mind.

Reply:
I did some git history digging, looks like it's 
https://github.com/nytimes/library/commit/bcb4cd0c19de711b54b94a79b6ab57fd71bd623e#diff-04c97f3a78a1f0cbdcd4754ca986bbafb55bb3072dac3b3df1f2dcbccbdb7f5cR19

seems like it's an app functionality triggered on the backend by simply setting the `edit=1` query parameter. so yeah just a link should be fine!